hypha/apply/projects/admin_views.py

```diff
@@ -21,3 +21,11 @@ class CreateProjectSOWFormView(CreateProjectApprovalFormView):
 
 class EditProjectSOWFormView(EditProjectApprovalFormView):
     pass
+
+
+class CreateProjectReportFormView(CreateProjectApprovalFormView):
```

Comment:
Reports are created by applicants/vendors so should not be admin related in any way.

Reply:
This is the report form, though, not the report itself. The admins set up the form they way they would like the applicants to see it.

Reply:
Perhaps other roles should be able to edit this besides admins, right? I have yet to get to the exact permissions.